project-addons/flask_middleware_connector/events/partner_events.py

```diff
@@ -105,6 +106,14 @@ def delay_export_partner_create(session, model_name, record_id, vals):
                              True):
                         export_rmaproduct.delay(session, 'claim.line', line.id,
                                                 priority=10, eta=240)
+
+            pickings = session.env['stock.picking'].search(
+                [('partner_id', 'child_of', partner.id)])
```

Comment:
Usando el operador child_of el siguiente elemento de la tupla tiene que ser una lista, aunque tenga un único elemento, pero lista, sino da un warning en el log, cada vez que lo ejecuta.

Reply:
Es posible que debas filtrar por estado también, los cancelados al menos no debería de enviarse, esto aplica a todos los search que haces abajo sobre stock.picking

Reply:
Por lo que he visto esa tabla no tiene commercial_partner_id asi que voy a cambiar el child of por un =

Reply:
No, está bien así, simplemente ponlo como lista, el child_of los usas para buscar sobre res.partner y esta si tiene child_of